docs/endpoints/intro.md

```diff
@@ -1,10 +1,10 @@
 # Introduction
 
-This document presents an overview of all the endpoints and IRISHub-SDK a node exposes.
+This document presents an overview of all the endpoints and IRIShub-SDK a node exposes.
```

Comment:
This document presents an overview of all the API endpoints a node exposes and the higher-level client SDKs.